tweetflask.py

```diff
@@ -97,18 +97,18 @@ def challenge(self):
 
     basic_auth = ChallengeAuth(app)
     with open('config.yaml', 'r') as yfile:
-        yamldata = yaml.load(yfile)
-    secureProtocol = yamldata['server']['secureProtocol']
+        yaml_data = yaml.load(yfile)
+    secureProtocol = yaml_data['server']['secureProtocol']
     try:
         context = ssl.SSLContext(getattr(ssl, secureProtocol))
     except AttributeError as e:
         exit('Invalid secureProtocol')
 
     context.load_cert_chain(
-        yamldata['server']['certPath'],
-        yamldata['server']['keyPath']
+        yaml_data['server']['certPath'],
+        yaml_data['server']['keyPath']
         )
-    app.config['BASIC_AUTH_USERNAME'] = yamldata['authentication']['username']
-    app.config['BASIC_AUTH_PASSWORD'] = yamldata['authentication']['password']
+    app.config['BASIC_AUTH_USERNAME'] = yaml_data['authentication']['username']
+    app.config['BASIC_AUTH_PASSWORD'] = yaml_data['authentication']['password']
 
-    app.run(debug=True, port=yamldata['server']['port'], ssl_context=context)
+    app.run(debug=True, port=yaml_data['server']['port'], ssl_context=context)
```

Comment:
`yaml_data['server']` and `yaml_data['authentication']` were used multiple times, could you store the value in a variable so that you won't need to look it up every time?